content/intro-to-storybook/react/en/screen.md

```diff
@@ -2,102 +2,135 @@
 title: 'Construct a screen'
 tocTitle: 'Screens'
 description: 'Construct a screen out of components'
-commit: '05aa2ef'
+commit: 'cec2e05'
 ---
 
 We've concentrated on building UIs from the bottom up, starting small and adding complexity. Doing so has allowed us to develop each component in isolation, figure out its data needs, and play with it in Storybook. All without needing to stand up a server or build out screens!
 
 In this chapter, we continue to increase the sophistication by combining components in a screen and developing that screen in Storybook.
 
-## Nested container components
+## Connected screens
 
-As our app is straightforward, the screen we’ll build is pretty trivial, simply wrapping the `TaskList` component (which supplies its own data via Redux) in some layout and pulling a top-level `error` field out of Redux (let's assume we'll set that field if we have some problem connecting to our server).
+As our app is straightforward, the screen we'll build is pretty trivial, simply fetching data from a remote API, wrapping the `TaskList` component (which supplies its own data from Redux), and pulling a top-level `error` field out of Redux.
 
-Let's start by updating our Redux store (in `src/lib/store.js`) to include the error field we want:
+We'll start by updating our Redux store (in `src/lib/store.js`) to connect to a remote API and handle the various states for our application (i.e., `error`, `succeeded`):
 
 ```diff:title=src/lib/store.js
- /* A simple redux store/actions/reducer implementation.
+/* A simple redux store/actions/reducer implementation.
  * A true app would be more complex and separated into different files.
  */
-import { configureStore, createSlice } from '@reduxjs/toolkit';
-
-+ // Our new error field is configured here
-+ const AppStateSlice = createSlice({
-+   name: "appState",
-+   initialState: "",
-+   reducers: {
-+     updateAppState: (state, action) => {
-+       return {
-+         ...state,
-+         isError: action.payload,
-+       };
-+     },
-+   },
-+ });
+import {
+  configureStore,
+  createSlice,
++ createAsyncThunk,
+} from '@reduxjs/toolkit';
 
 /*
  * The initial state of our store when the app loads.
- * Usually, you would fetch this from a server.
+ * Usually, you would fetch this from a server. Let's not worry about that now
  */
-const defaultTasks = [
-  { id: '1', title: 'Something', state: 'TASK_INBOX' },
-  { id: '2', title: 'Something more', state: 'TASK_INBOX' },
-  { id: '3', title: 'Something else', state: 'TASK_INBOX' },
-  { id: '4', title: 'Something again', state: 'TASK_INBOX' },
-];
+
+const TaskBoxData = {
+  tasks: [],
+  status: "idle",
+  error: null,
+};
+
+/*
+ * Creates an asyncThunk to fetch tasks from a remote endpoint.
+ * You can read more about it Redux Toolkit's thunks in the docs:
+ * https://redux-toolkit.js.org/api/createAsyncThunk
+ */
+
```

Comment:
A comment for a function doesn't usually have a blank line after it.

Reply:
Thanks for catching this. Fixed with it with the latest commit.